src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,109 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final double CAPACITY_INDEX = 1.5;
+    private static final int MAX_ARRAY = 10;
+    private T[] element;
```

Comment:
this is an array, so use plural for name
```suggestion
    private T[] elements;
```